gnmi/bgp/reconciler.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bgp
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+
+	"github.com/golang/glog"
+	"github.com/openconfig/lemming/gnmi/oc"
+	api "github.com/wenovus/gobgp/v3/api"
+	"github.com/wenovus/gobgp/v3/pkg/bgpconfig"
+	"github.com/wenovus/gobgp/v3/pkg/server"
+	"golang.org/x/net/context"
+)
+
+const (
+	gracefulRestart = false
+)
+
+type bgpReconciler struct {
```

Comment:
maybe merge this struct with bgpDeclTask, or is there a reason for them to be seperate?

Reply:
Merged structs.